packages/react-router-dom/docs/api/Link.md

```diff
@@ -81,6 +81,17 @@ let anchorRef = React.createRef()
 
 <Link to="/" innerRef={anchorRef} />
 ```
+## component: React.ComponentType&lt;any>
```

Comment:
React Router isn't written in TS (yet), so can we drop the type here and just say it's a React.Component?